monero/umbrel-app.yml

```diff
@@ -0,0 +1,27 @@
+manifestVersion: 1.1
+id: monero
+category: Finance
+name: Monero Node
+icon: https://raw.githubusercontent.com/deverickapollo/umbrel-monero/dev/ui/src/assets/community-monero-icon.png
+version: "0.18.2.2"
+tagline: Run your monero node and independently store and validate every single Monero transaction with it.
```

Comment:
Any ideas on a shorter tagline so that it can show up nicely in the app store? Maybe something as simple as `Run your personal Monero node`

<img width="370" alt="image" src="https://github.com/getumbrel/umbrel-apps/assets/85373263/13d87607-2ba6-466b-a3c5-ccaea656fa9b">